tests/realtikvtest/scripts/next-gen/bootstrap-test-with-cluster.sh

```
@@ -18,8 +18,13 @@
 # - pd: 2379, 2380, 2381, 2383, 2384
 # - tikv: 20160, 20161, 20162, 20180, 20181, 20182
 # - tikv-worker: 19000
+tiflash_compute_pid=""
+schema_manager_dir="/tmp/tidb-realtikvtest-schemas"
+
 function main() {
     local data_base_dir=$(mktemp -d)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)